aion/web/elm/src/Room/Models.elm

```diff
@@ -30,7 +30,8 @@ type alias UserList =
 
 
 type alias UserGameData =
-    { currentAnswer : String
+    {
+      currentAnswer : String
     }
```

Comment:
Please follow `elm-format`: https://github.com/avh4/elm-format
You can integrate it with your IDE and add watcher for files.
https://github.com/avh4/elm-format#editor-integration